tests/clickhouse/query_dsl/test_project_id.py

```diff
@@ -107,14 +107,23 @@
             "selected_columns": ["column1"],
             "conditions": [
                 [
-                    ["and", [["project_id", "=", 100], ["column1", "=", "something"]]],
+                    [
+                        "and",
+                        [
+                            ["equals", ["project_id", 100]],
+                            ["equals", ["column1", "'something'"]],
+                        ],
+                    ],
```

Comment:
What motivated this change?

Reply:
The query was invalid before for the AST so that test would fail now.
We cannot add snuba syntax conditions into functions.
The test was still working and relevant because it was a test for the old representation, so whether the ast parsing would work or not did not matter.